doc/sphinx/chpl2rst.py

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+"""chpl2rst converts a chapel program to an rst file, where all comments are
+rendered rst, and all code is wrapped in code blocks.
+
+Chapel files are converted as follows:
+
+* Line comments starting at the beginning of the line are converted to text
+* Block comments starting anywhere are converted to text
+* All other lines are wrapped as code-blocks
+
+example.chpl
+============
+
+/* This is
+     an example */
+proc foo() {
+    // this comment will be in the code block
+    var bar = 1; }
+// this comment will be text
+
+example.rst
+===========
+
+This is
+  an example
+
+.. code-block:: chapel
+
+    proc foo() {
+        // this comment will be in the code block
+        var bar = 1; }
+
+this comment will be text"""
+
+from __future__ import print_function
+
+import os
+import sys
+
+import argparse
+
+def get_arguments():
+    """
+    Get arguments from command line
+    """
+    parser = argparse.ArgumentParser(
+        prog='chpl2rst',
+        usage='%(prog)s  file.chpl [options] ',
+        description=__doc__,
+        formatter_class=argparse.RawDescriptionHelpFormatter)
+
+    parser.add_argument('chapelfiles', nargs='+',
+                        help='Chapel files to convert to rst')
+    parser.add_argument('--output', default='rst', choices=['stdout', 'rst'],
+                        help='destination of output')
+    parser.add_argument('--prefix', default='.',
+                        help='prefix path for output')
+    parser.add_argument('--codeblock', '-c', action='store_true', default=False,
+                        help='convert entire document into code block')
+    parser.add_argument('--verbose', '-v', action='store_true', default=False,
+                        help='verbosity')
+    return parser.parse_args()
+
+
+def chpl2rst(chapelfile):
+    """Convert Chapel program to restructured text"""
```

Comment:
Think this could use more details about what it actually does. i.e. turns multiline comments into regular text